commands/management/add_mysql_test.go

```diff
@@ -147,6 +148,23 @@ Table statistics collection disabled (always).
 	})
 }
 
+func TestCredentials(t *testing.T) {
+	data := `{"username": "testuser", "password": "testpass", "agentpassword": "testagentpass"}`
+	creds := commands.Credentials{}
+	cr, _ := CreateDummyCredentialsFile(data, "json", false)
+	ce, _ := CreateDummyCredentialsExecutable(data)
+
+	if err := creds.ReadFromSource(cr); err != nil {
+		t.Fatalf("failed to read from source: %v", err)
+	}
+	if err := creds.ReadFromSource(ce); err == nil {
+		t.Fatalf("unexpected exection of source: %v", err)
+	}
+	if creds.Username != "testuser" {
+		t.Fatalf("expected 'testuser', got: %v", creds.Username)
+	}
+}
+
```

Comment:
it doesn't look like this test should be part of this file

Reply:
I will move it to replace the base test